vizro-core/docs/pages/explanation/authors.md

```diff
@@ -10,7 +10,7 @@
 [Maximilian Schulz](https://github.com/maxschulz-COL),
 [Nadija Graca](https://github.com/nadijagraca),
 [Petar Pejovic](https://github.com/petar-qb)
-
+[Upekesha Ngugi](https://github.com/upekesha)
```

Comment:
Hey @upekesha - I'm just removing this change for now such that we can merge this PR without you having to push a changelog file :) I'll add you as a contributor to the section below in a separate PR, where we'll be adding all of the contributors from the GHC session 👍 

```suggestion
```